js/main.js

```diff
@@ -65,24 +76,42 @@ $nextButton.addEventListener('click', nextButtonEvent);
 
 function viewSwap(event) {
   if (event === 'globe') {
-    $globeContainer.className = 'container globe-container';
     $countryGenerator.className = 'container generator-container hidden';
     $listContainer.className = 'container list-container hidden';
+    $studyContainer.className = 'container study-container hidden';
+    $answerContainer.className = 'container answer-container hidden';
+    $globeContainer.className = 'container globe-container';
   } else if (event === 'country-generator') {
     $globeContainer.className = 'container globe-container hidden';
     $listContainer.className = 'container list-container hidden';
-    $countryGenerator.className = 'container generator-container';
     $listContainer.className = 'container list-container hidden';
+    $studyContainer.className = 'container study-container hidden';
+    $answerContainer.className = 'container answer-container hidden';
+    $countryGenerator.className = 'container generator-container';
   } else if (event === 'countries-list') {
     $globeContainer.className = 'container globe-container hidden';
     $countryGenerator.className = 'container generator-container hidden';
+    $studyContainer.className = 'container study-container hidden';
+    $answerContainer.className = 'container answer-container hidden';
     $listContainer.className = 'container list-container';
+  } else if (event === 'study-view') {
+    $globeContainer.className = 'container globe-container hidden';
+    $countryGenerator.className = 'container generator-container hidden';
+    $listContainer.className = 'container list-container hidden';
+    $answerContainer.className = 'container answer-container hidden';
+    $studyContainer.className = 'container study-container';
+  } else if (event === 'answer-view') {
+    $globeContainer.className = 'container globe-container hidden';
+    $countryGenerator.className = 'container generator-container hidden';
+    $listContainer.className = 'container list-container hidden';
+    $studyContainer.className = 'container study-container hidden';
+    $answerContainer.className = 'container answer-container';
   }
 }
```

Comment:
This function is getting pretty ridiculous, I would suggest going back to review `javascript-view-swapping` and refactor this to make use of a loop.